esp/src/src-react/components/DFSExists.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import { DefaultButton, DetailsList, DetailsListLayoutMode, IColumn } from "@fluentui/react";
+import { SizeMe } from "react-sizeme";
+import { csvParse } from "d3-dsv";
+import { DaliService, Result } from "@hpcc-js/comms";
+import { scopedLogger } from "@hpcc-js/util";
+import { TableGroup } from "./forms/Groups";
+import nlsHPCC from "src/nlsHPCC";
+import { HolyGrail } from "../layouts/HolyGrail";
+
+const logger = scopedLogger("src-react/components/DFSExist.tsx");
+
+const myDaliService = new DaliService({ baseUrl: "" });
+
+interface DFSExistProps {
+}
+
+export const DFSExists: React.FunctionComponent<DFSExistProps> = ({
+
+}) => {
+
+    const [columns, setColumns] = React.useState<IColumn[]>([]);
+    const [items, setItems] = React.useState<any[]>([]);
+    const [fileName, setFileName] = React.useState<string>("");
+
+    const onSubmit = React.useCallback(() => {
+        myDaliService.DFSExists({ FileName: fileName }).then(response => {
+            const data = csvParse(Result);
```

Comment:
I missed this in the previous PR, but that should be response.Result

Reply:
corrected for response, good catch.